src/nrf_802154.h

```diff
@@ -1020,10 +1044,19 @@ bool nrf_802154_ack_data_set(const uint8_t * p_addr,
                              uint8_t         data_type);
 
 /**
- * @brief Removes the address of a peer node for which the ACK data is set.
+ * @brief Removes the address of a peer node for which the ACK data is set from the pending bit list.
  *
  * The ACK data that was previously set for the given address is automatically removed.
  *
+ * The pending bit list works differently, depending on the upper layer for which the source
+ * address matching method is selected.
+ *
+ * For Thread, @ref NRF_802154_SRC_ADDR_MATCH_THREAD
+ * For Zigbee, @ref NRF_802154_SRC_ADDR_MATCH_ZIGBEE
+ * For Standard-compliant, @ref NRF_802154_SRC_ADDR_MATCH_ALWAYS_1
+ *
+ * Used method can be set during initialization phase by calling @ref nrf_802154_src_matching_method.

```

Comment:
```suggestion
* The method can be set during initialization phase by calling @ref nrf_802154_src_matching_method.
```
Mirror elsewhere in the file.